packages/ckeditor5-utils/src/keyboard.js

```diff
@@ -97,6 +102,7 @@ export function parseKeystroke( keystroke ) {
 
 	return keystroke
 		.map( key => ( typeof key == 'string' ) ? getCode( key ) : key )
+		.map( key => env.isMac && key == keyCodes.ctrl ? keyCodes.cmd : key )
```

Comment:
Yes. Otherwise, we would need to provide some "force" specifier or provide keystone alternative for MacOS.

Reply:
> Does this line means you cannot assign <kbd>Ctrl+...</kbd> keystroke on a Mac?

I'm worried about existing integrations here. Changing keystrokes in our features to resolve conflicts is a breaking change but disabling keystrokes in 3rd-party integrations is a whole new level. Just to clarify: there will be no way to use <kbd>Ctrl</kbd>+<kbd>whatever</kbd> on Mac after this change?

Reply:
After a talk with @Reinmar we decided that we will introduce `Ctrl!` as a forced `Ctrl` key that won't get mapped to `Cmd` on macOS. This is because some integrators or external features might want to use keystrokes that are not available after mapping to `Cmd` (for example `Cmd+Q`, `Cmd+T`, `Cmd+W`, etc.) but are available as `Ctrl+Q`, `Ctrl+T`, `Ctrl+W`.